src/flow_solver/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ set(FLOW_SOLVER_SOURCE
     flow_solver_cases/1d_burgers_viscous_snapshot.cpp
     flow_solver_cases/naca0012.cpp
     flow_solver_cases/gaussian_bump.cpp
+    flow_solver_cases/limiter_conv_tests.cpp
```

Comment:
```suggestion
    flow_solver_cases/limiter_convergence_tests.cpp
```

Reply:
reasonably verbose names make the code more readable

Reply:
Renamed the files to `limiter_convergence_tests`.